.travis.yml

```diff
@@ -150,9 +158,26 @@ after_success:
 after_failure:
 - dmesg | tail -100
 after_script:
-- if [ $TRAVIS_SECURE_ENV_VARS == true ]; then
+- if [ $TRAVIS_SECURE_ENV_VARS == true && $TRAVIS_EVENT_TYPE != cron ]; then
     $GCLOUD/gcloud components -q update gsutil;
     REPORT_PATH=$TRAVIS_BRANCH_$TRAVIS_JOB_NUMBER;
     $GCLOUD/gsutil -m cp -z html -z js -z xml -z css -r build/reports/tests gs://hellbender/test/build_reports/$REPORT_PATH/;
     echo "See the test report at https://storage.googleapis.com/hellbender/test/build_reports/$REPORT_PATH/tests/test/index.html";
   fi
+# This creates and uploads the gatk zip file to the nightly build bucket, only keeping the 10 newest entries
+- if [[ $TRAVIS_BRANCH == master && $TRAVIS_EVENT_TYPE == cron && $UPLOAD == true && "$(travis show master | head -2 | grep 'State:' | awk '{print $NF}')" == passed ]]; then
+    gsutil ls -l gs://gatk-nightly-builds | grep gatk | sort -r -k 2 | grep -o '\S\+$' | tail -n +11 | xargs -I {} gsutil rm {};
+    ./gradlew bundle;
+    $GCLOUD/gsutil -m cp build/"$(ls build/ | grep .zip)" gs://gatk-nightly-builds/"$(date +%Y-%m-%d)"-"$(ls build/ | grep .zip)";
+  fi
+#deploy:
```

Comment:
We could probably use the deploy blocks instead of our own gsutil installation for this upload and the test reports, but since we have to install gsutil for gatk tests anyway it never seemed worthwhile figuring out how to configure it.  Lets delete these commented out lines, but if you want to revive them later as a different mechanism for upload it might be worth looking at.